lib/one_model.ex

```diff
@@ -259,39 +266,38 @@ defmodule OneModel do
       Passing an atom will return the value of the given atom key.
       Passing a list of keys will return the default schema with the values for the specified keys.
       """
-      @spec get(id(), keyword()) :: t() | nil
+      @spec get(id(), keyword()) :: t() | nil | any()
```

Comment:
I don't like these being spec'ed as returning any(). If there is some value other than the t and nil values let's be more specific about what, and preferably justify why. On all of these.